src/styledConstants.ts

```diff
@@ -50,3 +50,11 @@ export const StyledContainer = styled.div`
   }
 `;
 
+export const StyledSectionSeparator = styled.div<{ $background: string }>`
```

Comment:
please leave comments in the constants file what elements you add, so someone else can understand and reuse it again.